src/draive/tools/toolbox.py

```
@@ -182,16 +182,23 @@ def tool_selection(
         else:
             return "auto"
 
-    def available_tools(self) -> Sequence[LMMToolSpecification]:
-        return [
+    def available_tools(
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)